vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb

```diff
@@ -0,0 +1,25 @@
+module SecureRooms
+
+  class OccupanciesController < ApplicationController
+
+    layout "plain"
+
+    before_action :load_occupancies
+
+    def index
+      @refresh_url = refresh_facility_secure_room_occupancies_url(current_facility, @secure_room.dashboard_token)
```

Comment:
Is there a way to collapse these two actions into one? Do something like `render ..., layout: !request.xhr?` so it has no layout when being fetched with ajax.